src/Bundle/test/src/Subscription/Entity/Subscription.php

```diff
@@ -27,23 +29,30 @@
 #[Resource(
     alias: 'app.subscription',
     section: 'admin',
+    formType: SubscriptionType::class,
     templatesDir: 'crud',
     routePrefix: '/admin',
 )]
 #[Index(grid: 'app_subscription')]
 #[Create]
 #[Update]
+#[ApplyStateMachineTransition(stateMachineTransition: 'accept')]
+#[ApplyStateMachineTransition(stateMachineTransition: 'reject')]
```

Comment:
Is it enough data to add such an operation? Don't we miss the graph here? How does it work?

Reply:
That graph is an optional value.

Reply:
On Winzou, the graph is called "default" is you do not specified it => $graph = $operation->getStateMachineGraph() ?? 'default';
here https://github.com/Sylius/SyliusResourceBundle/pull/612/files#diff-fccfca1acf785d9965e1c335fbcc071e6dffb7433489d6d2341b760ae68e2b4dR36

On Workflow, this is the "workflow name" and is value is null by default
https://github.com/Sylius/SyliusResourceBundle/pull/612/files#diff-52c7ef7ce600296cd01a16a35c3c604f6bc0ab7e8b8179138ad17111cf912267R36